index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <nav id="nav-flex-container">
+      <p>Welcome</p>
+      <img src="Images/Menu.svg" alt="Menu" />
+    </nav>
+    <header id="header-flex-container">
```

Comment:
- [ ] To make your html more [semantic](https://www.w3schools.com/html/html5_semantic_elements.asp), please kindly wrap this section in a `section` tag.

Reply:
Done.